core/src/com/unciv/ui/utils/CameraStageBaseScreen.kt

```diff
@@ -113,7 +113,10 @@ fun Button.enable() {
     color = Color.WHITE
     touchable = Touchable.enabled
 }
-
+var Button.isEnabled: Boolean
+    //Todo: Use in PromotionPickerScreen, TradeTable, WorldScreen.updateNextTurnButton
+    get() = touchable == Touchable.enabled
+    set(value) = if (value) enable() else disable()
```

Comment:
Hmmmmmmmmmm
I'm not sure if I'm for this or not

Reply:
Well, I think it improves readability on the calling site a lot. Probably gets inlined too. But - your call. I'm going to be unhappy either way, nothing to do with Unciv.

Reply:
Ahh fine
We can always remove it later if it's problematic